pipelines/broad/rna_seq/RNAWithUMIsPipeline.changelog.md

```diff
@@ -1,3 +1,11 @@
+# 1.0.8
+* Specify the RSEM post-processed transcriptome bam as output
```

Comment:
Please add the date / and date of last commit line here (above the list of changes).

Reply:
Done